resources/views/vendor/mail/study-invitation.blade.php

```diff
@@ -1,5 +1,5 @@
 @component('mail::message')
-{{ __('You have been invited to join the :study study!', ['study' => $invitation->study->name]) }}
+{{ __('You have been invited to join the study **:study** by :invitedBy as role :role', ['study' => $invitation->study->name, 'invitedBy' => $invitedBy,'role' => $invitation->role]) }}
```

Comment:
{{ __('You have been invited to join (role: :role) the study **:study** by :invitedBy', ['study' => $invitation->study->name, 'invitedBy' => $invitedBy,'role' => $invitation->role]) }}